packages/google_sign_in/test/google_sign_in_test.dart

```diff
@@ -327,6 +331,39 @@ void main() {
         ],
       );
     });
+
+    test('getAuthentication', () async {
+      await googleSignIn.signIn();
+      log.clear();
+
+      final GoogleSignInAccount user = googleSignIn.currentUser;
+      final GoogleSignInAuthentication auth = await user.getAuthentication();
+
+      expect(auth.accessToken, '456');
+      expect(auth.idToken, '123');
+      expect(
+        log,
+        <Matcher>[
```

Comment:
Nit: Is a list required when there's only one matcher? Ditto below

Reply:
Yea I think it should be a list because log is a list. It helps verify that only one `MethodCall` was invoked.